File: state-machine/src/main/java/br/com/leonardoferreira/poc/statemachine/Runner.java
package br.com.leonardoferreira.poc.statemachine;

import br.com.leonardoferreira.poc.statemachine.domain.enumeration.OrderEvents;
import br.com.leonardoferreira.poc.statemachine.domain.enumeration.OrderStates;
import br.com.leonardoferreira.poc.statemachine.service.OrderService;
import lombok.extern.slf4j.Slf4j;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.boot.ApplicationArguments;
import org.springframework.boot.ApplicationRunner;
import org.springframework.statemachine.StateMachine;
import org.springframework.stereotype.Component;

import java.math.BigDecimal;

@Slf4j
@Component
public class Runner implements ApplicationRunner {

    @Autowired
    private OrderService orderService;

    @Override
    public void run(ApplicationArguments args) throws Exception {
        log.info("----------------------------------------------------");
        Long orderId = orderService.create(new BigDecimal("32.43"), "BR-LEFE-001");

        StateMachine<OrderStates, OrderEvents> payStateMachine = orderService.pay(orderId);
        log.info(">> pay state: {}", payStateMachine.getState().getId());

        StateMachine<OrderStates, OrderEvents> fulfillStateMachine = orderService.fulfill(orderId);
        log.info(">> fulfill state: {}", fulfillStateMachine.getState().getId());

        StateMachine<OrderStates, OrderEvents> cancelStateMachine = orderService.cancel(orderId);
        log.info(">> cancel state: {}", cancelStateMachine.getState().getId());

        log.info("----------------------------------------------------");
    }

}

File: jpa/jpa-one-to-one/src/main/kotlin/br/com/leonardoferreira/poc/Runner.kt
package br.com.leonardoferreira.poc

import br.com.leonardoferreira.poc.domain.Customer
import br.com.leonardoferreira.poc.domain.CustomerAddress
import br.com.leonardoferreira.poc.repository.CustomerAddressRepository
import br.com.leonardoferreira.poc.repository.CustomerRepository
import org.springframework.boot.ApplicationArguments
import org.springframework.boot.ApplicationRunner
import org.springframework.stereotype.Component

@Component
class Runner(
    private val customerRepository: CustomerRepository,
    private val customerAddressRepository: CustomerAddressRepository
) : ApplicationRunner {

    override fun run(args: ApplicationArguments) {
        (1..15)
            .map { System.currentTimeMillis() + it }
            .forEach { sharedId ->
                val customer = Customer(
                    id = sharedId,
                    name = "customer $sharedId",
                    email = "customer$sharedId@email.com"
                )

                customerRepository.save(customer)

                val customerAddress = CustomerAddress(
                    id = sharedId,
                    street = "street $sharedId",
                    city = "city $sharedId",
                    number = "$sharedId"
                )

                customerAddressRepository.save(customerAddress)
            }
    }

}